.github/workflows/publish.yaml

```diff
@@ -0,0 +1,31 @@
+name: Publish Package
+on:
+  push:
+    tags:
```

Comment:
can we limit this to only be push tags on main?

Reply:
we could do 
```yaml
on:
  release: [published]
```